tests/project/test_util.py

```diff
@@ -231,3 +233,53 @@ def test_to_string_literal(raw_string, literal):
 )
 def test_escape_like_pattern(raw_string, escaped):
     assert escape_like_pattern(raw_string) == escaped
+
+
+@pytest.mark.parametrize(
+    "id1, id2, concatenated_value",
+    [
```

Comment:
we need agreement on this. For now, I only add quotes in the result if one of the 2 ids originally had quotes.

Reply:
yes corret.